documentation/modules/configuring/con-config-mirrormaker2-replication.adoc

```diff
@@ -81,6 +85,28 @@ Both topics enable replication to be fully restored from the correct offset posi
 
 MirrorMaker 2.0 uses its `MirrorCheckpointConnector` to emit _checkpoints_ for offset tracking.
 
+== Synchronizing consumer group offsets
+
+The `__consumer_offsets` topic stores information on committed offsets, according to consumer group.
+Offset synchronization periodically transfers the consumer offsets for the consumer groups of a source cluster into the consumer offsets topic of a target cluster.
```

Comment:
"Offset synchronization" is a MM2 concept, correct?

Reply:
Yes